src/main/java/io/cdap/plugin/gcp/bigquery/source/BigQueryAvroToStructuredTransformer.java

```diff
@@ -103,4 +104,9 @@ protected Object convertField(Object field, Schema fieldSchema) throws IOExcepti
     }
     return super.convertField(field, fieldSchema);
   }
+
+  @Override
+  protected Object convertBytes(Object field) {
```

Comment:
Where is this method being called, dont' see this in PR.

Reply:
It gets called here in Base class - `RecordConverter` which will get triggered from `super.convertField(field, fieldSchema)` method call in `transform` method of derived class -  `BigQueryAvroToStructuredTransformer.java`